plugins/ingest/src/main/java/org/elasticsearch/ingest/processor/set/SetProcessor.java

```diff
@@ -34,21 +35,25 @@
 
     public static final String TYPE = "set";
 
-    private final Map<String, Object> fields;
+    private final String fieldName;
+    private final Object value;
 
-    SetProcessor(Map<String, Object> fields) {
-        this.fields = fields;
+    SetProcessor(String fieldName, Object value) {
+        this.fieldName = fieldName;
+        this.value = value;
     }
 
-    Map<String, Object> getFields() {
-        return fields;
+    String getFieldName() {
```

Comment:
seems like we use field or fieldName throughout the different processors. Can we settle on the same name for all processors? I think field is enough.